babel-macro/src/test/macro.spec.ts

```diff
@@ -7,7 +7,7 @@ process.chdir(__dirname);
 pluginTester({
   plugin,
   babelOptions: {
-    filename: join(__filename, 'tested-file.js'),
+    filename: join(__dirname, 'babel-output', 'tested-file.js'),
```

Comment:
The tests are passing after this change?

Reply:
Yes, you can see the status on the PR.